tutorials/early_stopping/early_stopping.ipynb

```diff
@@ -0,0 +1,978 @@
+       "    val_acc  trial_index arm_name  hidden_size_1  hidden_size_2  \\\n",
+       "0    0.9691            0      0_0             55             66   \n",
+       "3    0.9296            1      1_0             20            121   \n",
+       "7    0.9405            2      2_0             33             87   \n",
+       "8    0.8749            3      3_0             22             21   \n",
+       "9    0.9753            4      4_0             65             27   \n",
+       "10   0.9680            5      5_0             70            104   \n",
+       "11   0.9517            6      6_0             49             53   \n",
+       "12   0.9768            7      7_0             80             60   \n",
+       "13   0.9641            8      8_0             42             27   \n",
+       "14   0.9733            9      9_0             75             34   \n",
+       "1    0.9736           10     10_0             62             48   \n",
+       "2    0.9633           11     11_0             87             36   \n",
+       "4    0.9752           12     12_0             84             97   \n",
+       "5    0.9735           13     13_0             72             41   \n",
+       "6    0.9772           14     14_0             98             34   \n",
+       "\n",
+       "    learning_rate   dropout   trial_status generation_method  \n",
+       "0        0.001961  0.041773      COMPLETED             Sobol  \n",
+       "3        0.000175  0.413174      COMPLETED             Sobol  \n",
+       "7        0.003465  0.180462      COMPLETED             Sobol  \n",
+       "8        0.004707  0.460226  EARLY_STOPPED             Sobol  \n",
+       "9        0.000476  0.067369      COMPLETED             Sobol  \n",
+       "10       0.002672  0.000000      COMPLETED              GPEI  \n",
+       "11       0.003229  0.000000  EARLY_STOPPED              GPEI  \n",
+       "12       0.000633  0.047332      COMPLETED              GPEI  \n",
+       "13       0.000439  0.042350  EARLY_STOPPED              GPEI  \n",
+       "14       0.001257  0.106674      COMPLETED              GPEI  \n",
+       "1        0.000359  0.096906      COMPLETED              GPEI  \n",
+       "2        0.000729  0.001421  EARLY_STOPPED              GPEI  \n",
+       "4        0.001076  0.083246      COMPLETED              GPEI  \n",
+       "5        0.000567  0.067441      COMPLETED              GPEI  \n",
+       "6        0.000286  0.086008      COMPLETED              GPEI  "
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "exp_to_df(experiment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a06172ba",
+   "metadata": {},
+   "source": [
+    "## Visualizations\n",
+    "\n",
+    "Finally, we show a visualization of learning curves versus actual elapsed wall time. This helps to illustrate that stopped trials make room for additional trials to be run."
```

Comment:
Maybe expand / elaborate a bit on this figure? 

I also wonder if there is a rough back of the envelope calculation we could do here with the result form this run to estimate what kinds of savings one might expect. A simple thing would be to make the assumption that the ES trials wouldn't have been the best and would also not have substantially affected the model, then we can give a rough ball park number for how much compute / wall time this would have saved relative to the full run (acknowledging the obvious flaws with this estimate).